### DISCUSSION TITLE
Error with docker installation and Page Editor

### DISCUSSION THREAD

**Discussion by LBoese** [givanz/Vvveb#113] [General]
First of all great work and I am really eager to get everything running. But...

I am running the vvveb/vvvebcms:latest on docker.

When I create a new page and select Open in Page Editor, I get the error message "Page not found!"

The Debug information is "Controller file not found!"

Thanks

**Comment by givanz** [comment]
Thanks for the bug report, it's fixed in the latest update https://github.com/givanz/Vvveb/commit/a7ed5bbb3f9926210e6ecb509a0950d0da48de03
